rom/laghos.cpp

```diff
@@ -134,6 +134,7 @@ int main(int argc, char *argv[])
     double blast_energy = 0.25;
     double blast_position[] = {0.0, 0.0, 0.0};
     double dt_factor = 1.0;
+    double windowTol = 0.99;
```

Comment:
Should we use the default value of 1? I am wondering if this will pass regression tests.